podcasts/PodcastTableViewCell.swift

```diff
@@ -0,0 +1,61 @@
+import SwiftUI
+import PocketCastsServer
+import PocketCastsDataModel
+
+final class PodcastTableViewCell: ThemeableCell {
+    static var reuseIdentifier: String = "PodcastTableViewCell"
+    private var viewModel: PodcastCellViewModel?
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        contentConfiguration = nil
+        viewModel = nil
+        if #available(iOS 16.0, *) {
+        } else {
+            contentView.subviews.forEach { $0.removeFromSuperview() }
+        }
+    }
+
+    func configure(with viewModel: PodcastCellViewModel) {
+        self.selectedStyle = .primaryUi02Active
+
+        self.viewModel = viewModel
+
+        if #available(iOS 16.0, *) {
+            self.contentConfiguration = UIHostingConfiguration {
+                PodcastTableCellView(viewModel: viewModel)
+                    .environmentObject(Theme.sharedTheme)
+            }
+        } else {
+            let view = PodcastTableCellView(viewModel: viewModel)
+            let uiView = view.environmentObject(Theme.sharedTheme).uiView
+            contentView.addSubview(uiView)
```

Comment:
I think you need to add constraints here in order for the `uiView` resize correctly inside the `contentView` you may also need to disable `translatesAutoresizingMaskIntoConstraints` in the uiView.